localkube/kubelet.go

```diff
@@ -33,25 +33,25 @@ var (
 	KubeletStop    chan struct{}
 )
 
-func NewKubeletServer(clusterDomain, clusterDNS string) Server {
+func NewKubeletServer(clusterDomain, clusterDNS string, containerized bool) Server {
 	return &SimpleServer{
 		ComponentName: KubeletName,
-		StartupFn:     StartKubeletServer(clusterDomain, clusterDNS),
+		StartupFn:     StartKubeletServer(clusterDomain, clusterDNS, containerized),
 		ShutdownFn: func() {
 			close(KubeletStop)
 		},
 	}
 }
 
-func StartKubeletServer(clusterDomain, clusterDNS string) func() {
+func StartKubeletServer(clusterDomain, clusterDNS string, containerized bool) func() {
 	KubeletStop = make(chan struct{})
 	config := options.NewKubeletServer()
 
 	// master details
 	config.APIServerList = []string{APIServerURL}
 
 	// Docker
-	config.Containerized = true
+	config.Containerized = containerized
```

Comment:
Under this code there's: `config.ResolverConfig = "/rootfs/etc/resolv.conf"`
The `/rootfs` under should be dynamically changed based on this variable